tests/playwright/sanity/modules/playing-cards/widget.test.js

```diff
@@ -0,0 +1,68 @@
+import { test, expect } from '@playwright/test';
+import WpAdminPage from '../../../pages/wp-admin-page';
+import EditorSelectors from '../../../selectors/editor-selectors';
+
+test.describe( 'Playing Cards @playing-cards', () => {
```

Comment:
@pkniazevych Well done on this test, I like it! I've suggested a few tweaks that you can consider as you write tests.